src/node_crypto.cc

```diff
@@ -4556,6 +4575,43 @@ bool ECDH::IsKeyPairValid() {
 }
 
 
+struct CryptoJob : public ThreadPoolWork {
+  Environment* const env;
```

Comment:
There’s already `env_` on the `ThreadPoolWork` class, could we just expose that instead?

Reply:
Not as a method, I tried.  Diamond problem: it conflicts with env() from AsyncWrap / BaseObject.